R/configuration-plan.R

```diff
@@ -160,8 +160,7 @@ ConfigurationPlan <- R6::R6Class(
           # With recursive=TRUE, overwrite images
           file.copy(
             from = file.path(inputContentSubFolder, fileName),
-            to = file.path(outputContentSubFolder, fileName),
-            recursive = TRUE
```

Comment:
this is suspicious here. Why don't we copy recursively? Do we only have one layer?

Reply:
I had two issues using `file.copy`
1- I wanted to use `recursive` directly but it creates a subfolder instead of copying only the content
2- If the argument `from` of `file.copy` was a folder, it was trying to copy it as file and it did not work properly (which was the initial warning of issue #736 )

To fix both, I now use `recursive` in `list.files` and `list.dirs` to fetch the list of **all** files and folders. 
This way, I can create all the directories recursively from the folder names first and then copy directly the files at appropriate locations